iaso/api/org_unit_change_request_configurations/views.py

```diff
@@ -0,0 +1,106 @@
+import django_filters
+from django.db.models import Q
+from rest_framework import viewsets, filters, serializers
+from rest_framework.decorators import action
+from rest_framework.response import Response
+
+from iaso.api.org_unit_change_request_configurations.filters import OrgUnitChangeRequestConfigurationListFilter
+from iaso.api.org_unit_change_request_configurations.pagination import OrgUnitChangeRequestConfigurationPagination
+from iaso.api.org_unit_change_request_configurations.serializers import (
+    OrgUnitChangeRequestConfigurationListSerializer,
+    OrgUnitChangeRequestConfigurationRetrieveSerializer,
+    OrgUnitChangeRequestConfigurationWriteSerializer,
+    OrgUnitChangeRequestConfigurationUpdateSerializer,
+    OrgUnitTypeNestedSerializer,
+    OrgUnitChangeRequestConfigurationAuditLogger,
+    ProjectIdSerializer,
+)
+from iaso.models import OrgUnitChangeRequestConfiguration, OrgUnitType, Project
+
+
+class OrgUnitChangeRequestConfigurationViewSet(viewsets.ModelViewSet):
```

Comment:
Is there a specific reason you are not deriving from iaso.api.commom.ModelViewSet ?

Reply:
I was told that the `OrgUnitChangeRequest` API was the most up to date and to do something similar to this one. It doesn't inherit from `iaso.api.common.ModelViewSet` :man_shrugging: